zenpy/lib/endpoint.py

```diff
@@ -211,7 +211,7 @@ class SearchEndpoint(BaseEndpoint):
 
     """
 
-    ZENDESK_DATE_FORMAT = "%Y-%m-%d"
+    ZENDESK_DATE_FORMAT_NAIVE = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
This breaks other places that reference this name, eg https://github.com/Shaakmiller/zenpy/blob/da6af2cc5a5d83f9ce8a965306a9a6b69c717405/zenpy/lib/endpoint.py#L223. I think it's best to leave this constant as is as many parts of the Zendesk API only expect dates.

It turns out that the IS08601 format is already defined here - https://github.com/Shaakmiller/zenpy/blob/da6af2cc5a5d83f9ce8a965306a9a6b69c717405/zenpy/lib/endpoint.py#L67. Maybe extract this into a top level constant and use it in the format between code?

Reply:
I see that my version of Zenpy does not have that additional reference that you mentioned, I must be using an older version or something. 

So I reinstated ZENDESK_DATE_FORMAT as it was, and I saw that your constant ISO_8601_FORMAT was in PrimaryEndpoint which inherits from BaseEndpoint. SearchEndpoint also inherits from BaseEndpoint so I transferred ISO_8601_FORMAT into BaseEndpoint so both classes can use it.

I tested it and it seems to work just fine. 

Let me know if I may have overlooked something.

Regards,